uber/automated_emails.py

```diff
@@ -28,12 +28,12 @@
                 ident='attendee_badge_confirmed')
 
 AutomatedEmail(Group, '{EVENT_NAME} group payment received', 'reg_workflow/group_confirmation.html',
-         lambda g: g.amount_paid == g.cost and g.cost != 0,
+         lambda g: g.amount_paid == g.cost and g.cost != 0 and g.leader_id,
```

Comment:
Don't send the group confirmation email if the leader has not been assigned yet.